src/lua/zencode_w3c.lua

```diff
@@ -86,29 +86,27 @@ local function export_verification_method(doc)
     return res
 end
 
+local function decode_jwt_parts(s)
+    if type(s) == 'string' then
+        return O.from_string(s)
+    elseif type(s) == 'number' then
+        if s==math.floor(s) and s >= 1500000000 and s < 2000000000 then
+            return U.new(s)
```

Comment:
What we though during this alias creation was U as unix timestamp, but it is ok for me to stick to TIME to be more explicit

Reply:
Also I am inclined to move this check as a method inside the TIME class, something like `TIME.istimestamp(s)` so that we keep it all into a single place in case we need to change or refine it in the future.

Reply:
When we check for the boundaries is beacuse we are autodetecting timestamps. Thus I called the function `TIME.is_autodetected_time()`